pandas/core/generic.py

```diff
@@ -3583,8 +3583,8 @@ def replace(self, to_replace=None, value=None, inplace=False, limit=None,
               wrappers around the scipy interpolation methods of similar
               names. These use the actual numerical values of the index. See
               the scipy documentation for more on their behavior
-              `here <http://docs.scipy.org/doc/scipy/reference/interpolate.html#univariate-interpolation>`__  # noqa
-              `and here <http://docs.scipy.org/doc/scipy/reference/tutorial/interpolate.html>`__  # noqa
```

Comment:
I *think* this will give a problem with out linter (that's the reason for the "#noqa"), but can wait on travis to confirm. 
A way how I have solved this in other places is to ensure there are no spaces on the line that is too long (then the linter makes an exception). So you can do:

```
    ... `here
    <http://docs.scipy.org/doc/scipy/reference/interpolate.html#univariate-interpolation>`__ 
```

and then it does not complain about the too long line (this is also how it is done in some of the 'Notes' sections below